athina/steps/llm.py

```
@@ -268,3 +268,77 @@ def execute(self, input_data: dict, **kwargs) -> str:
+    async def execute_async(self, input_data: dict, **kwargs) -> str:
+        """Execute a prompt with the LLM service asynchronously."""
+        if input_data is None:
+            input_data = {}
+
+        if not isinstance(input_data, dict) and self.input_key:
+            raise ValueError("PromptExecution Error: Input data must be a dictionary")
+
+        try:
+            messages = self.template.resolve(**input_data)
+            # Convert messages to API format
+            api_formatted_messages = [msg.to_api_format() for msg in messages]
```

Ruff (0.8.2)

Local variable `api_formatted_messages` is assigned to but never used

Remove assignment to unused variable `api_formatted_messages`

(F841)